### PR TITLE
phASER-30666e7, MACS2-2.1.1.20160309

### PR DESCRIPTION
**Pull request by npklein**

### CODE REVIEW

File: easybuild/easyconfigs/m/MACS2/MACS2-2.1.1.20160309-foss-2015b-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+pyshortver = '.'.join(pyver.split('.')[0:2])
+pylibdir = 'lib/python%s/site-packages' % pyshortver
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('numpy', '1.10.4', versionsuffix),
+]
+
+md5sums = ['1cc2fab44b232992060c7ab592bf43fa']
```
Comment (pneerincx) [inline easybuild/easyconfigs/m/MACS2/MACS2-2.1.1.20160309-foss-2015b-Python-2.7.11.eb]: md5sums -> checksums

File: easybuild/easyconfigs/p/phASER/phASER-30666e7.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+builddependencies = (
+    ('Cython','0.22-foss-2015b-Python-2.7.9'),
+)
+
+
+
+dependencies = [('SAMtools','1.2-foss-2015b'),
```
Comment (pneerincx) [inline easybuild/easyconfigs/p/phASER/phASER-30666e7.eb]: Do not put toolchain in version suffix. E.g.:

('SAMtools','1.2-foss-2015b'), -> ('SAMtools', '1.2' , '', ('foss', '2015b')),

('PyVCF', '0.6.7-foss-2015b-Python-2.7.9'), -> ('PyVCF', '0.6.7', '-Python-2.7.9', ('foss', '2015b')),

etc.

**Comment by npklein** [comment]
@pneerincx @freerkvandijk can the changes be reviewed?

**Comment by npklein** [comment]
@pneerincx changes added

**Pull request merged**